src/Apps/Notifications/NotificationsApp.tsx

```diff
@@ -1,27 +1,68 @@
-import { FullBleed } from "@artsy/palette"
 import { MetaTags } from "Components/MetaTags"
 import { Notifications } from "Components/Notifications/Notifications"
 import { createFragmentContainer, graphql } from "react-relay"
 import { NotificationsApp_me$data } from "__generated__/NotificationsApp_me.graphql"
+import { Media } from "Utils/Responsive"
+import { NotificationsContextProvider } from "Components/Notifications/useNotificationsContext"
+import { NotificationQueryRenderer } from "Components/Notifications/Notification"
+import { useRouter } from "found"
+import { GridColumns, Column, Flex } from "@artsy/palette"
+import { DESKTOP_NAV_BAR_HEIGHT } from "Components/NavBar/constants"
+
+const DESKTOP_HEIGHT = `calc(100vh - ${DESKTOP_NAV_BAR_HEIGHT}px)`
+const MIN_LIST_WIDTH = 340
 
 interface NotificationsAppProps {
   me: NotificationsApp_me$data
 }
 
-const NotificationsApp: React.FC<NotificationsAppProps> = props => {
-  const { me } = props
+const NotificationsApp: React.FC<NotificationsAppProps> = ({ me }) => {
+  const { match } = useRouter()
+
+  const isNotificationsPage = match.location.pathname === "/notifications"
 
   return (
-    <>
+    <NotificationsContextProvider>
       <MetaTags title="Notifications | Artsy" pathname="/notifications" />
 
-      <FullBleed>
-        <Notifications
-          mode="page"
-          unreadCounts={me.unreadNotificationsCount ?? 0}
-        />
-      </FullBleed>
-    </>
+      <Media greaterThan="xs">
+        <GridColumns>
+          <Column
+            span={3}
+            borderRight="1px solid #ddd"
+            minWidth={MIN_LIST_WIDTH}
+          >
+            <Flex height={DESKTOP_HEIGHT} flexDirection="column">
+              <Notifications
+                mode="page"
+                unreadCounts={me.unreadNotificationsCount ?? 0}
+              />
+            </Flex>
+          </Column>
+
+          <Column span={9}>
+            <Flex
+              flexDirection="column"
+              height={DESKTOP_HEIGHT}
+              overflow="auto"
+            >
+              <NotificationQueryRenderer />
+            </Flex>
+          </Column>
+        </GridColumns>
+      </Media>
+
+      <Media at="xs">
+        {isNotificationsPage ? (
+          <Notifications
+            mode="page"
+            unreadCounts={me.unreadNotificationsCount ?? 0}
+          />
+        ) : (
+          <NotificationQueryRenderer />
+        )}
```

Comment:
Although this might not be the cleanest solution, it has some advantages:

I'm using the component `NotificationsApp` for both routes: `/notifications` and `/notification/$notificationId`. This allows for navigation from `/notifications` to `/notification/$notificationId` (by clicking on an element in the Activity list in the split view) without the Activity list reloading, as both routes render the same fragment container.

I'm interested in hearing your thoughts about the solution!